jekyll/_sass/_sidebar.scss

```diff
@@ -66,13 +66,13 @@
   .nav-item {
     text-transform: uppercase;
     font-size: $font-size-micro;
-    font-weight: 100;
+    font-weight: 600;
     margin: 10px 0;
     letter-spacing: .5px;
   }
   .subnav-item {
     font-weight: 100;
-    margin: 5px 0;
+    margin: 5px 0 5px $spacing-medium-horizontal;
```

Comment:
I like seeing these in variables! Should `5px` be as well?

Reply:
the problem is that there are lots of bespoke values in this design/layout; not being designed with a specific grid in place means spacing values aren't very clean/reusable. 

So I can create variables here, but they would be just to improve semantics, and I'm not sure how valuable that would be. For this instance, where we are using 5px a lot, I'll add a variable, though.